sdk/keyvault/Azure.Security.KeyVault.Certificates/src/CertificateClient.cs

```diff
@@ -28,7 +28,7 @@ public CertificateClient(Uri vaultUri, TokenCredential credential)
 
         public CertificateClient(Uri vaultUri, TokenCredential credential, CertificateClientOptions options)
         {
-            _vaultUri = vaultUri ?? throw new ArgumentNullException(nameof(credential));
+            _vaultUri = vaultUri ?? throw new ArgumentNullException(nameof(vaultUri));
```

Comment:
>_vaultUri = vaultUri ?? throw new ArgumentNullException(nameof(vaultUri)); [](start = 12, length = 74)

You can revert this change I have fixed this in the CertificateClient implementation which I should PR this afternoon

Reply:
If it'll help avoid merge conflicts, I certainly will.